tools/rna_tools/pipmir/tool-data/twobit_update.loc.sample

```diff
@@ -0,0 +1,23 @@
+#This is a sample file distributed with Galaxy that is used by some
```

Comment:
thanks. lastz_seqs.loc.sample has three columns, though. I just wonder what the standards applied when these location files were prepared. for 2bit, there are already 2-column versus 3-cloumn format. 

2-column: <Build>	<FullPathToFile>
3-column: <unique_build_id>	<display_name>		<file_path>

I made a 4-column version which worked on our server: <unique_build_id>   <dbkey>   <display_name>   <file_path>

it seems to me there is no rule to govern the preparation of loc file.